docs/beamrunner/beamrunner-5-ref.md

```diff
@@ -55,7 +56,8 @@ The Streams Runner package contains the following folders:
     </tr>
     <tr>
       <td><code class="highlighter-rouge">jarsToStage</code></td>
-      <td>A list of JAR files (separated by colons) that are required to run the Apache Beam application. Include the JAR files that contain your program and any dependencies. (You don’t need to include Beam Google IO SDK or core Beam JAR files.) The listed JAR files are added to the SAB file.<br /><br /><strong>Note</strong>: The use of fat or uber JAR files increases the size of the SAB file and can negatively impact submission times to IBM Cloud.</td>
+      <td>A list of JAR files (separated by colons) that are required to run the Apache Beam application. Include the JAR files that contain your program and any dependencies. (You don’t need to include Beam Google IO SDK or core Beam JAR files.) The listed JAR files are added to the SAB file.<br /><br /><strong>Note</strong>: The use of fat or uber JAR files can reduce the number of JARs needed to be specified, but care should be taken not to include redundant JARs provided by the Streams

```

Comment:
Change
The use of fat or uber JAR files can reduce the number of JARs needed to be specified, but care should be taken not to include redundant JARs provided by the Streams...
to
The use of fat or uber JAR files can reduce the number of JAR files that must be specified, but take care not to include JAR files that are provided by the Streams...